sdk/loadtestservice/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (Unreleased)
+
+- First stable release of Azure Load Testing client library for Java with single top-level `LoadTestingClientBuilder` which can build `LoadTestAdministration` and `LoadTestRun` clients.
```

Comment:
We can drop the phrase after "with single top-level..." in the CHANGELOG. This should be documented in the README.

Reply:
Removed